feconf.py

```diff
@@ -796,6 +796,7 @@ def get_empty_ratings():
 ROLE_ID_COLLECTION_EDITOR = 'COLLECTION_EDITOR'
 ROLE_ID_MODERATOR = 'MODERATOR'
 ROLE_ID_ADMIN = 'ADMIN'
+ROLE_ID_TOPIC_MANAGER = 'TOPIC_MANAGER'
```

Comment:
Put this in the appropriate place in the hierarchical order.

Reply:
Done.